gugudan.java

```diff
@@ -0,0 +1,18 @@
+import java.util.Scanner;
+public class gugudan {
+    public static void main(String[] args) {
+        System.out.println("구구단 중 출력할 단은? ");
+        Scanner sc = new Scanner(System.in);
+        int input = sc.nextInt();
+        System.out.println("사용자가 입력한 단: " + input + "단");
+        if (input < 2) {
```

Comment:
여기서 논리 연산자인 OR을 이용하시면 else if 없이 코드 길이를 줄일 수 있습니다!
```suggestion
if (input < 2 || input > 9) 
```
한번 해보세요! ㅎ.ㅎ 고생하셨습니다~